access/types/forms.py

```diff
@@ -323,26 +320,29 @@ def create_more(self, configuration):
 
     def create_choices(self, configuration):
         '''
-        Creates field choices by configuration.
-
+        Creates choices for dropdown, select, radio and checkbox-type questions.
+        Create also the lists of correct, neutral and initially selected options.
         '''
         choices = []
         initial = []
         correct = []
+        neutral = []
         if "options" in configuration:
             i = 0
             for opt in configuration["options"]:
                 label = opt.get('label', "")
                 value = self.option_name(i, opt)
                 choices.append((value, mark_safe(label)))
+                if opt.get('correct') == "neutral":
+                    neutral.append(value)
                 if opt.get('correct', False) is True:
```

Comment:
`elif` could be used since `opt.get('correct')` can not be both `"neutral"` and `True` at the same time. Now, two `if` conditions are always checked.

Reply:
You are correct! I changed that now